### PR TITLE
[Polkadot Wiki Migration] Node Endpoints Page

### PR DESCRIPTION
**Pull request by CrackTheCode016**
> **This page is from the [Polkadot Wiki](https://wiki.polkadot.network/docs/maintain-endpoints), and will be heavily edited, updated, and appropriated as needed.**

Page referenced: https://wiki.polkadot.network/docs/maintain-endpoints

### CODE REVIEW

File: develop/application-devs/interact/endpoints.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use.  These endpoints can be used for development, RPC, or other purposes in which network access is required.
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:3]:
```suggestion
description: List of node endpoints and examples of use. These endpoints can be used for development, RPC, or other purposes for which network access is required.
```

File: develop/application-devs/interact/endpoints.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+Ideally, one may run their own node when interacting with the
+[Polkadot network](https://polkadot.network/) via [Polkadot-JS Apps](https://polkadot.js.org/apps/)
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:7]:
```suggestion
[Polkadot network](https://polkadot.network/) via [Polkadot.Js Apps](https://polkadot.js.org/apps/)
```

File: develop/application-devs/interact/endpoints.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+or other UIs and programmatic methods. Another option would be to connect to one of the several
+public endpoints provided by infrastructure and API service providers. For development convenience,
+[Parity Tech](https://www.parity.io/) maintains archive nodes for Polkadot, Kusama, and their test
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:10]: It seems that there are multiple line breaks in this paragraph. Could you please review the formatting of this paragraph as well as the others?

File: develop/application-devs/interact/endpoints.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+networks with public endpoints. These endpoints can be used with
+[Polkadot-JS API](https://polkadot.js.org/docs/api) to interact with their respective chains. The
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:12]:
```suggestion
[Polkadot.Js API](https://polkadot.js.org/docs/api) to interact with their respective chains. The
```

File: develop/application-devs/interact/endpoints.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+tables below list these endpoints.
+
+### Network Endpoints
+
+Endpoints for all production and test networks are listed on the
+[Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts) which are accessed from
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:18]:
```suggestion
[Polkadot.Js UI](https://polkadot.js.org/apps/#/accounts) which are accessed from
```

File: develop/application-devs/interact/endpoints.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use.  These endpoints can be used for development, RPC, or other purposes in which network access is required.
+---
+
+Ideally, one may run their own node when interacting with the
+[Polkadot network](https://polkadot.network/) via [Polkadot-JS Apps](https://polkadot.js.org/apps/)
+or other UIs and programmatic methods. Another option would be to connect to one of the several
+public endpoints provided by infrastructure and API service providers. For development convenience,
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:9]:
```suggestion
public endpoints provided by the infrastructure and API service providers. For development convenience,
```

File: develop/application-devs/interact/endpoints.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+[Polkadot-JS API](https://polkadot.js.org/docs/api) to interact with their respective chains. The
+tables below list these endpoints.
+
+### Network Endpoints
+
+Endpoints for all production and test networks are listed on the
+[Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts) which are accessed from
+[here](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints).
+Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo test network are
+maintained by the community. System Chains as well as Westend and Rococo test network endpoints
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:21]:
```suggestion
maintained by the community. System Chains, as well as Westend and Rococo test network endpoints
```

File: develop/application-devs/interact/endpoints.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+maintained by Parity Technologies are listed below (with the exception of [Paseo](https://github.com/paseo-network), which is maintained by the wider community):
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:22]:
```suggestion
maintained by Parity Technologies, are listed below (with the exception of [Paseo](https://github.com/paseo-network), which is maintained by the wider community):
```

File: develop/application-devs/interact/endpoints.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+
+=== "Polkadot"
+    | Network      | WSS Endpoint                               |
+    | ------------ | ------------------------------------------ |
+    | Asset Hub    | wss://polkadot-asset-hub-rpc.polkadot.io   |
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:28]:
```suggestion
    | Asset Hub    | [wss://polkadot-asset-hub-rpc.polkadot.io](wss://polkadot-asset-hub-rpc.polkadot.io){target=\_blank}   |
```

I'd add this reference so users can easily copy and paste the endpoint. Not sure if this is the best way to handle though

File: develop/application-devs/interact/endpoints.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+### Third Party Providers
+
+There are a number of third-party providers of RPC infrastructure to the Polkadot and Kusama
+communities, commonly providing access to multiple networks and parachains in a single service. They
+provide additional services such as higher rate limits, potentially more reliable and scalable
+service, and additional metrics.
+
+!!! note
+    The list of third party RPC endpoints above for Polkadot and Kusama is directly fetched from
+    [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer)
```
Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:59]:
```suggestion
    [Polkadot.Js UI](https://polkadot.js.org/apps/#/explorer)
```

Comment (nhussein11) [inline develop/application-devs/interact/endpoints.md:59]: All those links should include {target=\_blank}, like: 

[Polkadot.Js UI](https://polkadot.js.org/apps/#/explorer){target=\_blank}

Comment (CrackTheCode016) [reply develop/application-devs/interact/endpoints.md:59]: We probably need to start addressing the renaming soon I guess too? https://forum.polkadot.network/t/polkadot-developers/9788

 For now we can just use the current terms though.

File: develop/application-devs/interact/endpoints.md (hunk 3 above)
Comment (CrackTheCode016) [reply develop/application-devs/interact/endpoints.md]: Yes, comes from copying from the Wiki which has a specific formatter. WIll review and ensure formatting is normal

File: develop/application-devs/interact/endpoints.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use. These endpoints can be used for development, RPC, or other purposes for which network access is required.
+---
+
+Ideally, one may utilize their own node when interacting with the [Polkadot network](https://polkadot.network/){target=_blank} via [Polkadot Js UI Apps](https://polkadot.js.org/apps/){target=_blank} or other user interfaces and programmatic methods. Another option would be to connect to one of the several public endpoints provided by infrastructure and API service providers. For development convenience, [Parity Tech](https://www.parity.io/){target=_blank} maintains archive nodes for Polkadot, Kusama, and their test networks with public endpoints. These endpoints can be used with [Polkadot Js UI API](https://polkadot.js.org/docs/api){target=_blank} to interact with their respective chains.
```
Comment (0xLucca) [inline develop/application-devs/interact/endpoints.md:6]:
```suggestion
Ideally, one may utilize their own node when interacting with the [Polkadot network](https://polkadot.network/){target=\_blank} via [Polkadot.js UI Apps](https://polkadot.js.org/apps/){target=\_blank} or other user interfaces and programmatic methods. Another option would be to connect to one of the several public endpoints provided by infrastructure and API service providers. For development convenience, [Parity Tech](https://www.parity.io/){target=\_blank} maintains archive nodes for Polkadot, Kusama, and their test networks with public endpoints. These endpoints can be used with [Polkadot.js UI API](https://polkadot.js.org/docs/api){target=\_blank} to interact with their respective chains.
```

Fix other occurrences of Polkadot.js and add `\` to the `_blank` attribute

File: develop/application-devs/interact/endpoints.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+### Network Endpoints
+
+Endpoints for all production and test networks are listed on the [Polkadot Js UI](https://polkadot.js.org/apps/#/accounts){target=_blank} which are accessed from [here](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints){target=_blank}. Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo test network are maintained by the community. System Chains as well as Westend and Rococo test network endpoints maintained by Parity Technologies are listed below (with the exception of [Paseo](https://github.com/paseo-network){target=_blank}, which is maintained by the wider community):
```
Comment (0xLucca) [inline develop/application-devs/interact/endpoints.md:10]: It is not a good practice to use `here` for a link. Replace with a more self-descriptive word

**Comment by dawnkelly09** [comment]
@erin this is ready for you to review. I did a little reformat to make the endpoints copy paste friendly as well as some formatting and grammar stuff.

File: develop/application-devs/interact/endpoints.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+## Network Endpoints
+
+Endpoints for all production and TestNets are listed on the [Polkadot.js UI](https://polkadot.js.org/apps/#/accounts){target=\_blank} which accesses them from the [endpoints directory](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints){target=\_blank} of the Polkadot.js repository. Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo TestNet are maintained by the community. System Chains and Westend and Rococo TestNet endpoints maintained by Parity Technologies are listed below. Note the exclusion of [Paseo](https://github.com/paseo-network){target=\_blank}, which is maintained by the wider community.
+
+
+=== "Polkadot"
+
+    ``` text title="Asset Hub"
+        wss://polkadot-asset-hub-rpc.polkadot.io
```
Comment (eshaben) [inline develop/application-devs/interact/endpoints.md:20]: The indent can be removed from all of these

File: develop/application-devs/interact/endpoints.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+        wss://paseo-rpc.dwellir.com
+    ```
+
+## Third Party Providers
+
+There are a number of third-party providers of RPC infrastructure to the Polkadot and Kusama communities, commonly providing access to multiple networks and parachains in a single service. They provide additional services such as higher rate limits, potentially more reliable and scalable service, and additional metrics.
+
+!!! note
+    The list of third party RPC endpoints for Polkadot and Kusama is directly fetched from
+    [Polkadot.js UI](https://polkadot.js.org/apps/#/explorer){target=\_blank}
```
Comment (eshaben) [inline develop/application-devs/interact/endpoints.md:70]:
```suggestion
    The list of third party RPC endpoints for Polkadot and Kusama is directly fetched from [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank}.
```

File: develop/application-devs/interact/endpoints.md [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+
+## Network Endpoints
+
+Endpoints for all production and TestNets are listed on the [Polkadot.js UI](https://polkadot.js.org/apps/#/accounts){target=\_blank} which accesses them from the [endpoints directory](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints){target=\_blank} of the Polkadot.js repository. Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo TestNet are maintained by the community. System Chains and Westend and Rococo TestNet endpoints maintained by Parity Technologies are listed below. Note the exclusion of [Paseo](https://github.com/paseo-network){target=\_blank}, which is maintained by the wider community.
+
+
+=== "Polkadot"
```
Comment (eshaben) [inline develop/application-devs/interact/endpoints.md:17]: This looks nice

File: develop/application-devs/interact/.pages [hunk 16]
```diff
@@ -1,3 +1,4 @@
 title: Interact with the Network
 nav:
   - index.md
+  - endpoints.md
```
Comment (eshaben) [inline develop/application-devs/interact/.pages:4]:
```suggestion
  - 'Node Endpoints': 'endpoints.md'
```

**Review by eshaben — COMMENTED** [review]
@dawnkelly09 thanks for reformatting, looks nice ✨

**Comment by eshaben** [comment]
@dawnkelly09 please resolve merge conflicts too

File: develop/application-devs/interact/endpoints.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use. These endpoints can be used for development, RPC, or other purposes for which network access is required.
+---
+
+# Node Endpoints
+
+## Introduction
+
+One may utilize their own node when interacting with the [Polkadot network](https://polkadot.network/){target=\_blank} via [Polkadot.js API](https://polkadot.js.org/apps/){target=\_blank} or other user interfaces and programmatic methods. Another option would be to connect to one of the several public endpoints provided by infrastructure and API service providers. For development convenience, [Parity Tech](https://www.parity.io/){target=\_blank} maintains archive nodes for Polkadot, Kusama, and their TestNets with public endpoints. These endpoints can be used with [Polkadot.js API](https://polkadot.js.org/docs/api){target=\_blank} to interact with their respective chains.
```
Comment (eshaben) [inline develop/application-devs/interact/endpoints.md:10]:
```suggestion
One may utilize their own node when interacting with the [Polkadot network](https://polkadot.network/){target=\_blank} via [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} or other user interfaces and programmatic methods. Another option would be to connect to one of the several public endpoints provided by infrastructure and API service providers. For development convenience, [Parity Tech](https://www.parity.io/){target=\_blank} maintains archive nodes for Polkadot, Kusama, and their TestNets with public endpoints. These endpoints can be used with [Polkadot.js API](https://polkadot.js.org/docs/api){target=\_blank} to interact with their respective chains.
```

**Review by eshaben — COMMENTED** [review]
One last thing!

**Review by eshaben — APPROVED** [review]
LGTM!

**Review by nhussein11 — APPROVED** [review]
LGTM!

**Pull request merged**